src/types/item.schema.js

```diff
@@ -0,0 +1,26 @@
+gql`
```

Comment:
I'm getting `ReferenceError: gql is not defined` here. Does that run for you without the import? It also doesn't explicitly export anything.